portfolio/src/main/webapp/index.html

```diff
@@ -36,15 +37,15 @@ <h2>About Me... </h2>
         </p>
     </div>
 
-    <!-- place the cool image hover effect of each (for each) -->
+    place the cool image hover effect of each (for each)
```

Comment:
Is this supposed to be uncommented?

Reply:
It looks like you're doing this a lot, this will make each one of these comments display in the html.